src/Components/Grants/GrantsShow.jsx

```diff
@@ -63,45 +43,31 @@ export default function GrantsShow() {
   const totalWordCount = countTotalSectionsWords(grant?.sections);
   const [overview, setOverview] = useState(false);
 
-  // const sensors = useSensors(
-  //   useSensor(PointerSensor)
-  //   // This breaks forms nested under drag and drop! The space key triggers
-  //   // this sensor. TODO: Circle back to this!
-  //   // useSensor(KeyboardSensor, {
-  //   //   coordinateGetter: sortableKeyboardCoordinates,
-  //   // })
-  // );
-
-  const heroButtons = useCallback(() => {
-    if (overview) {
-      return (
-        <Button variant="outlined" onClick={() => setOverview(!overview)}>
-          Content View
-        </Button>
-      );
-    } else
-      return (
-        <>
-          <Button
-            variant="outlined"
-            as={CurrentOrganizationLink}
-            to={`/grants/${grant.id}/copy/`}
-          >
-            Copy
-          </Button>
-          <Button
-            variant="outlined"
-            as={CurrentOrganizationLink}
-            to={`/grants/${grant.id}/edit/`}
-          >
-            Edit
-          </Button>
-          <Button variant="outlined" onClick={() => setOverview(!overview)}>
-            Overview
-          </Button>
-        </>
-      );
-  }, [overview, grant.id]);
+  const heroButtons = () => (
+    <>
+      <Button
+        variant="outlined"
+        as={CurrentOrganizationLink}
+        to={`/grants/${grant.id}/copy/`}
+      >
+        Copy
+      </Button>
+      <Button
+        variant="outlined"
+        as={CurrentOrganizationLink}
+        to={`/grants/${grant.id}/edit/`}
+      >
+        Edit
+      </Button>
+      <Button
+        variant="outlined"
+        as={CurrentOrganizationLink}
+        to={`/grants/${grant.id}/overview/`}
+      >
+        Overview
+      </Button>
+    </>
+  );
```

Comment:
not necessarily something that needs to change, but this doesn't actually need to be wrapped in a function unless we needed to render buttons based on some arguments

so it could be like:
```suggestion
  const heroButtons = (
    <>
      <Button
        variant="outlined"
        as={CurrentOrganizationLink}
        to={`/grants/${grant.id}/copy/`}
      >
        Copy
      </Button>
      <Button
        variant="outlined"
        as={CurrentOrganizationLink}
        to={`/grants/${grant.id}/edit/`}
      >
        Edit
      </Button>
      <Button
        variant="outlined"
        as={CurrentOrganizationLink}
        to={`/grants/${grant.id}/overview/`}
      >
        Overview
      </Button>
    </>
  );
```

then further down you'd just reference it like `{heroButtons}` instead of calling it (cause it's not a function anymore)